### ISSUE TITLE
Proposal: Add warning when changing payment's customer.

### ISSUE THREAD

**Issue by mintplugins** [easydigitaldownloads/easy-digital-downloads#5565]
When you change a payment's customer, it happens immediately and all seems correct. However, potential problems lie under the surface in relation to customer meta.  For an example see:
https://github.com/easydigitaldownloads/edd-recurring/issues/676

Another example: if you have customer meta from something like Checkout Fields Manager, one might (wrongfully) assume that information would be moved from Customer 1 to Customer 2 when you change the payment's customer. 

However, this isn't true (nor should it be). I propose we issue a warning to say something like "Are you sure you want to do this? Any information attached to the old customer will not be transferred to the new customer."

**Comment by chriscct7** [comment]
CFM will transfer that data if you tell it to save the data to the payment
not the user. Obviously CFM shouldn't transfer user meta.

**Comment by mintplugins** [comment]
Correct. I think we should just make customers aware of that as it may not be as obvious to them.

**Comment by chriscct7** [comment]
Yep, 100% agree. We should also consider doing so on the user edit screen, even though it's not an EDD page, as we update the customer email whenever the user email is changed
